### PR TITLE
update messages

### PR DESCRIPTION
**Pull request by h9-tect**
solving the issue in #150

### CODE REVIEW

File: opendevin/action/agent.py [hunk 1]
```diff
@@ -32,7 +33,8 @@ def run(self, controller: "AgentController") -> "Observation":
 
     @property
     def message(self) -> str:
-        return f"Thinking: {self.thought}"
+        return f"I'm pondering over the idea: '{self.thought}'. Give me a second to process this."
```
Comment (rbren) [inline opendevin/action/agent.py:36]: Let's just make this `self.thought`
```suggestion
        return self.thought
```

File: opendevin/action/agent.py [hunk 2]
```diff
@@ -44,7 +46,8 @@ def run(self, controller: "AgentController") -> "Observation":
 
     @property
     def message(self) -> str:
-        return f"Echoing: {self.content}"
+        return f"You said '{self.content}', right? Let me reflect that back to you. Here it is!"
```
Comment (rbren) [inline opendevin/action/agent.py:49]:
```suggestion
        return self.content
```

File: opendevin/action/agent.py (hunk 1 above)
Comment (rbren) [reply opendevin/action/agent.py:36]: actually--this should probably just be `self.content`, and we should remove `thought` as a variable

**Comment by h9-tect** [comment]
Thank you so much again @rbren , updated

**Review by rbren — APPROVED** [review]

**Pull request merged**